### PR TITLE
front: add trainSchedule v1/v2 switch

### LINKED ISSUE #7016: front: add a switch v1/v2 and setup components
### Description and goal

Add a switch from v1 to v2

Technical precisions:
- duplicate the following components: views/Scenario, views/ManageTrainSchedule, view/SimulationResults, Timetable, Itinerary, PowerRestrictionsSelector
- duplicate the osrdConf store
- when creating a scenario, we create a timetable first and then create the scenario with the new timetable

### Acceptance criteria

AC
- [ ] Have a switch v1/v2, which allow to show only scenario v1 or v2
- [ ] if v2 is activated, the created scenarii will be v2
- [ ] if a v2 scenario is opened, the page is the new simulation result page v2 which does not work for the moment (BUT has the same interface than V1, except for the margins tab and the powerRestrictionSelector which have disappeared)

### PR DESCRIPTION
**Pull request by clarani**
closes #7016

### CODE REVIEW

**Review by flomonster — APPROVED** [review]
Tested, it seems to work :)

File: front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 
 import { useOsrdConfSelectors } from 'common/osrdContext';
-import DisplayVias from 'modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayVias';
+
+import DisplayVias from '../DisplayVias';
```
Comment (Uriel-Sautron) [inline front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx]: You can use the absolute path.

File: front/src/modules/powerRestriction/components/PowerRestrictionsSelector.tsx [hunk 2]
```diff
@@ -14,10 +14,10 @@ import type { IntervalItem } from 'common/IntervalsEditor/types';
 import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
 import { useAppDispatch } from 'store';
 
-import NO_POWER_RESTRICTION from './consts';
-import displayPowerRestrictionIntervals from './helpers/displayPowerRestrictionIntervals';
-import { countWarnings, getPowerRestrictionsWarnings } from './helpers/powerRestrictionSelector';
-import type { PowerRestrictionWarning } from './types';
+import NO_POWER_RESTRICTION from '../consts';
```
Comment (Uriel-Sautron) [inline front/src/modules/powerRestriction/components/PowerRestrictionsSelector.tsx]: You can use the absolute paths

**Review by Uriel-Sautron — COMMENTED** [review]
Thank you for this PR

File: front/src/modules/powerRestriction/components/PowerRestrictionsSelector.tsx (hunk 2 above)
Comment (clarani) [reply front/src/modules/powerRestriction/components/PowerRestrictionsSelector.tsx]: Fixed :+1:

File: front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx (hunk 1 above)
Comment (clarani) [reply front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx]: Done :+1:

**Review by Uriel-Sautron — APPROVED** [review]

**Review by Caracol3 — APPROVED** [review]
lgtm

**Review by RomainValls — APPROVED** [review]
LGTM 👍

**Review by kmer2016 — APPROVED** [review]
LGTM

**Review by Math-R — APPROVED** [review]
lgtm

**Pull request merged**